presto-main/src/test/java/com/facebook/presto/operator/index/TestFieldSetFilteringRecordSet.java

```diff
@@ -34,15 +34,14 @@
 
 public class TestFieldSetFilteringRecordSet
 {
-    private static final TypeManager TYPE_MANAGER = new TypeRegistry();
-    private static final FunctionRegistry FUNCTION_REGISTRY = new FunctionRegistry(TYPE_MANAGER, new BlockEncodingManager(TYPE_MANAGER), new FeaturesConfig());
-
     @Test
     public void test()
     {
+        TypeManager typeManager = new TypeRegistry();
```

Comment:
Extract it to `setUp` method for consistency, regardless of the fact it's the only test in that class...